controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -614,49 +614,46 @@ private Scope getScope(final String scopeName) {
         return result;
     }
 
-    private CompletableFuture<List<AbstractMap.SimpleEntry<Double, Double>>> getSealedRanges(final String scopeName,
-                                                                                             final String streamName,
-                                                                                             final List<Integer> sealedSegments,
-                                                                                             final OperationContext context,
-                                                                                             final Executor executor) {
-        return FutureHelpers.allOfWithResults(
-                sealedSegments.stream()
-                        .map((Integer value) ->
-                                getSegment(scopeName, streamName, value, context, executor).thenApply(segment ->
-                                        new AbstractMap.SimpleEntry<>(
-                                                segment.getKeyStart(),
-                                                segment.getKeyEnd())))
-                        .collect(Collectors.toList()));
-    }
-
-    private CompletableFuture<Long> findNumSplits(String scopeName, String streamName, Executor executor) {
+    private CompletableFuture<Pair<Long, Long>> findNumSplitsMerges(String scopeName, String streamName, Executor executor) {
         return getScaleMetadata(scopeName, streamName, null, executor).thenApply(scaleMetadataList -> {
-            long size = scaleMetadataList.size();
-            long totalNumSplits = 0;
-
-            for (int i = 0, j = 1; j < size; j++) {
-                long countSplitsEventI = scaleMetadataList.get(i).getSegments().size();
-                long countSplitsEventJ = scaleMetadataList.get(j).getSegments().size();
-                totalNumSplits = (countSplitsEventI < countSplitsEventJ) ? (countSplitsEventJ - countSplitsEventI) : 0;
+            int size = scaleMetadataList.size();
+            long totalNumSplits = 0, totalNumMerges = 0;
+            List<Segment> segmentList1, segmentList2;
```

Comment:
Check the Oracle Java Conventions:
http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141270.html#18761

One declaration per line.